main.ts

```diff
@@ -165,8 +165,15 @@ async function resolveLocalFileLink(
       root += INDEX_FILE;
     }
   }
-  const relativePath = join(directory, root);
+
   try {
+    // Matches two or more consecutive forward slashes
+    const regex = /\/{2,}/;
+    if (regex.test(root)) {
```

Comment:
Why is this better than
```suggestion
    if (root.includes("//")) {
```
?

Reply:
That will allow this typo `../advanced///transformers`

Reply:
Clever.